include/libdnf/base/solver_problems.hpp

```diff
@@ -31,6 +31,9 @@ namespace libdnf::base {
 class SolverProblems {
 public:
     SolverProblems(const SolverProblems & src);
+    SolverProblems(SolverProblems && src) noexcept = default;
+    SolverProblems & operator=(const SolverProblems & src) = default;
+    SolverProblems & operator=(SolverProblems && src) noexcept = default;
```

Comment:
This is not the fix I was asking for. Blanket-adding all constructors and operators as `= default` is pointless and redundant.

And, once again, this should be squashed into the original commits, not added on top here.

Reply:
I tried one again, please take a look.